app/lib/actions/candlepin/consumer/auto_attach_subscriptions.rb

```diff
@@ -6,8 +6,8 @@ class AutoAttachSubscriptions < Candlepin::Abstract
           param :uuid, String
         end
 
-        def plan(system)
-          plan_self(:uuid => system.uuid)
+        def plan(subscription_facet)
```

Comment:
What about abstracting this to just passing in UUID? The `subscription_fact` concept doesn't seem like something this abstraction layer needs to know about.